doc: Doc updates for 2.1.3 release (#9511)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

   :local:
   :depth: 2

.. note::

>    You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`known_issues`.

Reviewer: Shouldn't it be `known issues for nRF Connect SDK v2.1.3`_  rather than :ref:`known_issues`?
```suggestion
   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`known_issues`.
```

Author: Not sure about this.

Build system
============

> .. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0

Reviewer 2: Is the nanopb issue valid for all releases since 1.3.0? @peknis , @tejlmand ?

Reviewer 3: no, it's only a valid issue from v2.0.0, cause before that nanopb - protoc was not part of the toolchain and uesrs at that time were requested to install this tool manually.

Reviewer 3: ```suggestion
.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

NCSIDB-816: nanopb/protoc not specified correctly by the |NCS| Toolchain
  The |NCS| Toolchain includes the nanopb/protoc tool when installed, but the path to the tool is not propagated correctly to the |NCS| build system.

>   ** Workaround:** Find the location of the protoc executable in the |NCS| and manually add its path to your system's global path.

Reviewer 4: ```suggestion
  ** Workaround:** Locate the nanopb ``generator-bin`` directory in your |NCS| Toolchain, and add its path to your system's global path. This allows the protoc executable to be found.
```

Build system
============

.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0

> 

Reviewer 3: Somewhere in this section add the following known issue:
```suggestion
NCSIDB-816: nanopb / protoc not specified correctly by the nRF Connect SDK Toolchain.
  nRF Connect SDK Toolchain includes nanopb / protoc when installed but the path to the tool is not propagated correctly to the nRF Connect SDK build system.
  
  ** Workaround** Find the location of the protoc executable in the nRF Connect SDK and manually its path your system's global path
```

Author: Thanks. Is this fixed in NCS 2.2.0? If not, I'll also add the tag for it in the PR I'll open to update main.

Reviewer 3: No, it's not fixed in v2.2.0
